adguardhome.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "adguardhome",
+    "release": "12.2-RELEASE",
+    "artifact": "https://github.com/leandroscardua/iocage-plugin-adguardhome.git",
+    "official": false,
+    "pkgs": [
+        "adguardhome"
+    ],
+    "properties": {
+        "dhcp": 1,
+        "boot": "on"
```

Comment:
I think `boot` should be on by default so shouldn't be needed here. See same comment e.g. here: https://github.com/ix-plugin-hub/iocage-plugin-index/pull/244#discussion_r668820031

Reply:
Ok, I`m going to remove it.